go.mod

```diff
@@ -15,9 +17,11 @@ require (
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/google/go-cmp v0.6.0
 	github.com/google/go-github/v58 v58.0.0
+	github.com/gruntwork-io/go-commons v0.17.1
+	github.com/gruntwork-io/terragrunt v0.55.0
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/hc-install v0.6.1
-	github.com/hashicorp/hcl/v2 v2.14.1
+	github.com/hashicorp/hcl/v2 v2.17.0
```

Comment:
Was the version bump needed for tgdeps?

Reply:
That's version needed by Terragrunt, check here: https://github.com/gruntwork-io/terragrunt/blob/master/go.mod#L18

As both are semver compatible, AFAIK, it's not possible to use multiple versions in different packages in Go without replace directive hacks...

Reply:
I also had to bump toolchain version because of Terragrunt..